### DISCUSSION TITLE
Is is possible to run the GNNExplainer for a node regression task?

### DISCUSSION THREAD

**Discussion by alzaia** [pyg-team/pytorch_geometric#7809] [Q&A] [answered]
It seems that all the examples are based on either node classification or link prediction. Is GNNExplainer compatible with regression predictions? Thanks.

**Comment by rusty1s** [comment, marked as answer]
Yes, it is. We don't have an example for it, just a test though (see https://github.com/pyg-team/pytorch_geometric/blob/master/test/explain/algorithm/test_gnn_explainer.py#L190).
